docs/content/en/docs/packages/overview.md

```diff
@@ -12,7 +12,14 @@ weight: 1
 Components of EKS Anywhere curated packages consist of a controller, a CLI, and artifacts.
 
 ### Package controller
-The package controller will install, upgrade, configure and remove packages from the cluster. The package controller will watch the packages and packagebundle custom resources for the packages to run and their configuration values. The package controller only runs on the management cluster and manages packages on the management cluster and on the workload clusters.
+The package controller will install, upgrade, configure and remove packages from the cluster.
+
+The package controller will watch the packages and packagebundle custom resources to determine which packages to run and their appropriate configuration values.
```

Comment:
I feel like this says the same thing as the previous sentence but emphasizes the monitoring part. Does it make senes to drop it?